src/nft/ZoraCreator1155Impl.sol

```diff
@@ -556,15 +556,15 @@ contract ZoraCreator1155Impl is
     /// @param ids token ids to move
     /// @param amounts amounts of tokens
     /// @param data data of tokens
-    function _beforeTokenTransfer(
+    function _beforeBatchTokenTransfer(
         address operator,
         address from,
         address to,
         uint256[] memory ids,
         uint256[] memory amounts,
         bytes memory data
     ) internal override {
-        super._beforeTokenTransfer(operator, from, to, ids, amounts, data);
+        super._beforeBatchTokenTransfer(operator, from, to, ids, amounts, data);
```

Comment:
was this a part of the upstream openzeppelin upgrade?

Reply:
this was not, and i'm not exactly sure why because they make the distinction with `_doSafeTransferAcceptanceCheck` `_doSafeBatchTransferAcceptanceCheck` hooks on every transfer anyways?

it makes single token transfers cheaper bc we won't have to create fixed-size arrays, but if you'd prefer aligning closer with OZ that makes sense and i'll remove it

Reply:
[eg. This removes the need for lines 272-273](https://github.com/OpenZeppelin/openzeppelin-contracts-upgradeable/blob/dd8ca8adc47624c5c5e2f4d412f5f421951dcc25/contracts/token/ERC1155/ERC1155Upgradeable.sol#L268-L318)